util/ci

```diff
@@ -49,7 +49,7 @@ end
 case ARGV
 when %w(before_script)
   if TOOL.rubygems?
-    run('gem', %W(uninstall executable-hooks gem-wrappers bundler-unload -x --force -i #{`gem env home`.strip}@global))
+    run('ruby', %W(-I lib bin/gem uninstall executable-hooks gem-wrappers bundler-unload -x --force -i #{`gem env home`.strip}@global))
```

Comment:
Why is this change needed here?

Reply:
Not really "needed" but our build against `ruby-head` is [hitting precisely this bug](https://travis-ci.org/rubygems/rubygems/jobs/519464636#L236-L241). So by changing this, we get to use this fix, and thus get it green.

Reply:
Maybe have this in a separate PR? just so this change doesn't get obscure in the purpose of this PR, what you think?

Reply:
Sure.

Reply:
I removed the commit fixing the ruby-head build and will open a separate PR once this one is merged.

Reply:
Maybe we can merge the other one first and then rebase this one? so this is green in ruby-head too :)

Reply:
We need the fix in this PR as a prerequisite to fix ruby-head!

Reply:
If we want this PR to be green in ruby-head, I need to restore the commit I removed.

Reply:
That's why I put both commits in this PR, because they are very related.